ui/src/components/WMWorkerPoolEditor/index.jsx

```diff
@@ -300,37 +294,36 @@ export default class WMWorkerPoolEditor extends Component {
     return (
       <Fragment>
         <ErrorPanel fixed error={error} />
-        <List className={classes.list}>
-          <ListSubheader>Worker Pool ID *</ListSubheader>
-          <ListItem>
-            <TextField
-              name="workerPoolId1"
-              error={validation.workerPoolId1.error}
-              onChange={this.handleInputChange}
-              fullWidth
-              value={workerPool.workerPoolId1}
-              required
-              disabled={!isNewWorkerPool}
-              autoFocus={isNewWorkerPool}
-              helperText={validation.workerPoolId1.message}
-            />
-            <Typography className={classes.separator} variant="h5">
-              /
-            </Typography>
-            <TextField
-              name="workerPoolId2"
-              error={validation.workerPoolId2.error}
-              onChange={this.handleInputChange}
-              fullWidth
-              value={workerPool.workerPoolId2}
-              required
-              disabled={!isNewWorkerPool}
-              helperText={validation.workerPoolId2.message}
-            />
-          </ListItem>
-        </List>
-
-        <List className={classes.list}>
+        <List>
+          <div className={classes.workerPoolIdDiv}>
+            <ListItem>
+              <TextField
+                name="workerPoolId1"
+                label="Worker Pool ID 1"
```

Comment:
I think we can keep the old look for the worker pool. Having to write "1" and "2" is not the best :)

![Screen Shot 2019-10-10 at 2 43 23 PM](https://user-images.githubusercontent.com/3766511/66597157-63827480-eb6c-11e9-9d22-98a3c2bc3c91.png)